### PR TITLE
[Test-1] add test for NamedThreadFactory

### PR DESCRIPTION
**Pull request by xiaoheiAh**
### Ⅰ. Describe what this PR did
add test for NamedThreadFactory

### Ⅱ. Does this pull request fix one issue?

fix #195 

### Ⅲ. Why don't you add test cases (unit test/integration test)? 
already add

### Ⅳ. Describe how to verify it
run ut first

### CODE REVIEW

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ */
+public class NamedThreadFactoryTest {
+
+    @Test
+    public void testNewThread() {
+        NamedThreadFactory namedThreadFactory = new NamedThreadFactory("testNameThread", 5);
+
+        Thread testNameThread = namedThreadFactory
+            .newThread(() -> System.out.println(Thread.currentThread().getName()));
+        System.out.println(testNameThread.toString());
```
Comment (slievrly) [inline common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: remove System.out

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ * @date 2019/2/26
+ */
+public class NamedThreadFactoryTest {
+
+    @Test
+    public void testNewThread() {
+        NamedThreadFactory namedThreadFactory = new NamedThreadFactory("testNameThread", 5);
+
+        Thread testNameThread = namedThreadFactory
+            .newThread(() -> System.out.println(Thread.currentThread().getName()));
```
Comment (jovany-wang) [inline common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: Don't println() to stdout.

It make more sense here to use `assert`.

Comment (xiaoheiAh) [reply common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: already remove.

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java (hunk 1 above)
Comment (xiaoheiAh) [reply common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: already remove

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    @Test
+    public void testNewThread() {
+        NamedThreadFactory namedThreadFactory = new NamedThreadFactory("testNameThread", 5);
+
+        Thread testNameThread = namedThreadFactory
+            .newThread(() -> {
+                try {
+                    Thread.sleep(1000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
```
Comment (jovany-wang) [inline common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: Don't use `e.printStackTrace();` here.
We should declare the `throws` for this method:
```Java
public void testNewThread() throw InterruptedException {
```

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+public class NamedThreadFactoryTest {
+
+    @Test
+    public void testNewThread() {
+        NamedThreadFactory namedThreadFactory = new NamedThreadFactory("testNameThread", 5);
+
+        Thread testNameThread = namedThreadFactory
+            .newThread(() -> {
+                try {
+                    Thread.sleep(1000);
```
Comment (jovany-wang) [inline common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: Better to use `TimeUnit.second.sleep()`

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * @author Otis.z
+ * @date 2019/2/26
+ */
+public class NamedThreadFactoryTest {
+
+    @Test
+    public void testNewThread() {
+        NamedThreadFactory namedThreadFactory = new NamedThreadFactory("testNameThread", 5);
```
Comment (jovany-wang) [inline common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: private static final String XXX = "testNameThread";

File: common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java (hunk 3 above)
Comment (xiaoheiAh) [reply common/src/test/java/com/alibaba/fescar/common/thread/NamedThreadFactoryTest.java]: thx.i will be fixed on next commit

**Pull request merged**